packages/api/tests/lib/rules/test-rulesHelpers.js

```diff
@@ -1318,6 +1319,9 @@ test.serial('deleteSnsTrigger throws more detailed ResourceNotFoundError', async
     state: 'ENABLED',
   }));
 
+  const lambdaStub = sinon.stub(awsServices.lambda(), 'send');
+  lambdaStub.throws(error);
```

Comment:
Different pattern but based on what we're testing I think it makes sense to stub the whole`send`.